packager/react-packager/src/DependencyResolver/Package.js

```diff
@@ -90,4 +91,8 @@ class Package {
   }
 }
 
+function getReplacements (pkg) {
```

Comment:
May want to do a more robust check below that looks whether pkg[react-native] == null so that it returns empty strings when specified